app/bundles/CampaignBundle/Config/config.php

```diff
@@ -174,6 +174,9 @@
             ],
             'mautic.campaignrange.type.action' => [
                 'class' => 'Mautic\CampaignBundle\Form\Type\EventType',
+                /** CAPTIVEA.CORE START **/
+                'arguments' => 'mautic.factory',
```

Comment:
Let's not use MauticFactory in the new code since it had been deprecated. It will have to be changed in other places too.